style.css

```diff
@@ -512,3 +525,253 @@ a {
 .social-icons li {
   padding: 8px;
 }
+
+/* Here is where the desktop css starts */
+@media (min-width: 600px) {
+  #menu-btn {
+    display: none;
+  }
```

Comment:
- [ ] Min-width property on desktop should be in range of 992px - 1100px.

Reply:
I have updated the min-width parameters.